docs/advanced/scheduling/redeliver.md

```diff
@@ -1,6 +1,6 @@
 # Scheduled Redelivery
 
-> This page may be removed in the future, for more detail on handling exceptions is available at the [Handling exceptions](usage/exception.md) page.
+> This page may be removed in the future, for more detail on handling exceptions is available at the [Handling exceptions](../../usage/exception.md) page.
```

Comment:
just use `/usage/exception` - no relative paths, and you don't need .md with VuePress.

Reply:
thanks, I just put in `npm run docs:dev` to test this all locally before publishing this draft PR. Quite a nice experience, watching the files and rebuilding as you edit.

Reply:
What? did the docs get re-published from this PR? Because the links that were dead are no longer dead.